src/operators/doc_embedding_operator.rs

```diff
@@ -0,0 +1,35 @@
+use sqlx::{Pool, Postgres};
+
+use crate::{data::models::DocEmbedding, errors::ServiceError};
+
+pub async fn upsert_doc_embedding_query(
```

Comment:
Should prob keep the names of the two functions distinct between this and the qdrant op

Reply:
Fixex